test/utils/utilities_test.py

```diff
@@ -85,3 +85,43 @@ def test_returns_decoded_payload(self):
         )
 
         self.assertEqual(self.payload, decoded_payload)
+
+
+class TestSanitizeShopDomain(TestCase):
+    def test_returns_hostname_for_good_shop_domains(self):
+        good_shop_domains = [
+            "my-shop",
+            "my-shop.myshopify.com",
+            "http://my-shop.myshopify.com",
+            "https://my-shop.myshopify.com",
+        ]
+        sanitized_shops = [sanitize_shop_domain(shop_domain) for shop_domain in good_shop_domains]
+
+        self.assertTrue(all(shop_domain is not None for shop_domain in sanitized_shops))
```

Comment:
should we replace this line with what we expect to be returned?

Reply:
Does the latest change help clarify the expected output?

`self.assertTrue(all(shop == "my-shop.myshopify.com" for shop in sanitized_shops))`
and 
`self.assertTrue(all(shop == "my-shop.myshopify.io" for shop in sanitized_shops))`

Reply:
yeah that's perfect!